src/utils/apis/ProcessAPI.js

```diff
@@ -40,6 +40,19 @@ export const processAPI = {
                 stages: items
             })
         })
+    },
+
+    changeEndDate: function(id, date) {
+        const preparedDate = ("0" + date.getDate()).slice(-2) + "." + ("0" + (date.getMonth()+1)).slice(-2) + "." + date.getFullYear()
```

Comment:
To trochę wygląda jak magia, ale jak jest otestowana to przyjmuję XD

Reply:
Ręcznie otestowana, ale fakt, pasowałoby jakiś test do tego napisać XDDD